Jupyter Notebook Data Curation Primer/Jupyter Notebooks Data Curation Primer.md

```diff
@@ -100,10 +100,10 @@ The following elements outline recommendations for repositories accepting Jupyte
 - Additional files to request:
   - PDF of the Jupyter Notebook (export from Jupyter web application or [nbviewer](https://nbviewer.jupyter.org/))
   - reST export of the Jupyter Notebook (export from Jupyter web application)
-  - CodeMeta.json
-  - CITATION.cff
+  - CodeMeta.json, requirements.txt, or environment.yml (dependencies)
+  - CITATION.cff (a software citation file appropriate if not depositing in a repository)
```

Comment:
Adding a citation file is always appropriate— many repositories do not have the fields necessary to automatically generate a proper software citation.